pipeline.yml

```diff
@@ -3,18 +3,18 @@ env:
   - SLACK_QUANTUM_BUILD_CHANNEL : "#dynamo-jenkinsbuild"
   - SLACK_QUANTUM_BUILD_CREDENTIAL_ID : "slack-notify-token"
   - MAIL_QUANTUM_BUILD_RECIPIENT : "dynamo.plaid.dev@autodesk.com"
+  - ARTIFACTORY_BOBCAT_DOCKER_REGISTRY : "artifactory.dev.adskengineer.net"
   - TEST_REPORT_FORMAT : "junit"
 
 check_changelog_updated_on_pr: false
 pipeline_os: "Linux"
 force_slack_notification : true
-language: typescript
 nodejs_version: "10"
 
 build:
   customized_builder_image:
     registry: artifactory
-    url: autodesk-docker.art-bobcat.autodesk.com/quantum/hardened-nodejs:latest
+    url: artifactory.dev.adskengineer.net/docker-local-v2/docker-local-v2/dynamo/dynamolibrariejs:latest
```

Comment:
What's up with the path to the image ?
`/docker-local-v2/docker-local-v2` 
@jesusHCG @mjkkirschner  ?
seems like this works too `artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariej`

Reply:
i used the url from the artifactory

![image](https://user-images.githubusercontent.com/48030764/65341131-4ffb7380-db95-11e9-83d9-aeedf2ff4939.png)